chainreg/chainregistry.go

```diff
@@ -840,3 +859,37 @@ var (
 		},
 	}
 )
+
+// checkOutboundPeers checks the number of outbound peers connected to the
+// provided RPC client. If the number of outbound peers is below 6, a warning
+// is logged. This function is intended to ensure that the chain backend
+// maintains a healthy connection to the network.
+func checkOutboundPeers(cfg *Config, client *rpcclient.Client) error {
+	// On local test networks we usually don't have multiple
+	// chain backend peers, so we can skip that test.
+	if cfg.Bitcoin.SimNet || cfg.Bitcoin.RegTest {
+		return nil
+	}
+
```

Comment:
one last nit: 

I think we should put this check in the HealthCheck call back func instead of adding it here. That way when I read the call to `checkOutboundPeers` then can just assume that that is what it will do & dont need to go into the method itself to go see that it will only actually do that if the network is mainnet.